### ISSUE TITLE
convertAll is not working

### ISSUE THREAD

**Issue by haluk** [uniVocity/univocity-parsers#14]
I have the following code to convert "T" to 1.0 and "F" to 0.0. However, convertAll method does not convert the values but convertFields does.

```java
            CsvParserSettings parserSettings = new CsvParserSettings();
            parserSettings.setLineSeparatorDetectionEnabled(true);
            parserSettings.setHeaderExtractionEnabled(true);

            ObjectColumnProcessor objectColumnProcessor = new ObjectColumnProcessor();
            BooleanStringDoubleConversion conversion = new BooleanStringDoubleConversion();
            conversion.setTrueValues(new String[]{"T"});
            conversion.setFalseValues(new String[]{"F"});
//            objectColumnProcessor.convertFields(conversion).set("G56898");
            objectColumnProcessor.convertAll(conversion);

            parserSettings.setRowProcessor(objectColumnProcessor);

            CsvParser parser = new CsvParser(parserSettings);
            parser.parse(new FileReader("test.csv"));
            Map<String, List<Object>> columnValues = objectColumnProcessor.getColumnValuesAsMapOfNames();
            System.out.println(columnValues.keySet());
            for (Map.Entry<String, List<Object>> entry : columnValues.entrySet()) {
                String key = entry.getKey();
                List<Object> value = entry.getValue();
                System.out.println(key + "\t" + value);
            }

```

**Comment by jbax** [comment]
Fixed in [this commit](https://github.com/uniVocity/univocity-parsers/commit/2d8671c26ec1af7e3b5c0dd24ab07617edc81844)

Will part of release version 1.6.0

**Comment by jbax** [comment]
Released version [1.5.1](http://oss.sonatype.org/content/repositories/releases/com/univocity/univocity-parsers/1.5.1/univocity-parsers-1.5.1.jar) which includes this bugfix.

It may take a few hours until the artifact is available through the maven central repository. Use this link to get the latest release: http://oss.sonatype.org/content/repositories/releases/com/univocity/univocity-parsers/1.5.1/univocity-parsers-1.5.1.jar

Thank you for using our parsers!
